src/odoc/bin/main.ml

```diff
@@ -403,45 +403,46 @@ end = struct
   let targets = Targets.(cmd, info)
 end
 
-module Odoc_uri : sig
+let reference =
```

Comment:
Even if its duplicated, I think this definition belongs more to the inside of `Odoc_html_uri` and `Odoc_latex_uri`